.github/workflows/linters.yml

```
@@ -78,7 +78,7 @@ jobs:
             echo
             echo 'ERROR: Please run:'
             echo
-            echo '  CLANG_FORMAT_START="$(git merge-base HEAD <target-branch-name>)" make format-cpp'
+            echo '  CLANG_FORMAT_START="$(git merge-base HEAD master)" make format-cpp'
```

This raises two questions/concerns fro me:

1. This also runs on backport PRs where the branch might be something else, doesn't it?
2. This hard-codes `master` which we're trying to avoid as we try to finish the move to `main` instead.

Will this work instead?

```suggestion
            echo "  CLANG_FORMAT_START="$\(git merge-base HEAD ${GITHUB_BASE_REF}\)" make format-cpp"
```

---

> 1. This also runs on backport PRs where the branch might be something else, doesn't it?

Nope, it doesn't. It only runs on PRs that target the master branch https://github.com/nodejs/node/blob/036c4d407b7372694f98a876a67f2e19f5c35a91/.github/workflows/linters.yml#L53.

> 2. This hard-codes `master` which we're trying to avoid as we try to finish the move to `main` instead.

> Will this work instead?

Worth a try, let's see if that works!

---

@Trott it worked 
https://github.com/nodejs/node/runs/6054537767?check_suite_focus=true#step:6:50
```
 CLANG_FORMAT_START=$(git merge-base HEAD master) make format-cpp
```
Thanks, PTAL!